tests/e2e/cli/test_cli.py

```diff
@@ -284,6 +284,10 @@ def test_otx_explain_e2e_cli(
     if ("_cls" not in task) and (task not in ["detection", "instance_segmentation"]):
         pytest.skip("Supported only for classification, detection and instance segmentation task.")
 
+    # TODO (Galina): revert back when torch issue fixed
+    if task == "instance_segmentation":
```

Comment:
Isn't it possible to run w/o deterministic when task is i-seg?

Reply:
It is possible, although without checking reference values. But probably not that beneficial. Saliency maps for MRCNN actually generated on the MAPI side, (which is covered by tests there I believe). Also this would increase the number of if-else branches in the test.
Let me know if you have any objections about it.